content/blog/elm-review-v2/index.md

```diff
@@ -0,0 +1,269 @@
+
+Example use-cases:
+
+- [Make sure the links in your `README.md` point to the right version](https://github.com/jfmengels/review-documentation/blob/master/src/Documentation/ReadmeLinksPointToCurrentVersion.elm)
+- Make sure the links in your `README.md` point to existing modules
+- Report invalid or badly formatted Markdown content
+
+## Visiting the comments and documentation
+
+With this version, you can also look at the contents of the comments or documentation of a function or module (To be honest, this was just me forgetting to add a visitor for it, it would have been an easy addition to v1).
+
+Example use-cases (most of these are on my todo list for [`jfmengels/review-documentation`](https://package.elm-lang.org/packages/jfmengels/review-documentation/latest/)):
+
+- Making sure that the `@docs` in the module documentation are always correct and up to date
+- Report links to non-existing or non-exposed modules, or even to invalid functions/section ids
+- Reporting duplicate Markdown sections
+- Report the usage of words like `TODO`
+
+## Adding helpers
+
+Some tasks, like targeting a specific function, are harder than they should be. For instance, if you want to forbid `Foo.bar`, you'll need to handle multiple ways that the function can be called and imported, which is tedious and error-prone.
+
+I started writing a helper named [`elm-review-scope`](https://github.com/jfmengels/elm-review-scope) that deals with this problem, and makes some tasks as easy as they should be.
+
+## Much faster, and with a watch mode
+
+Version 1 focused on usability and on validating that `elm-review` was a good solution to the problems it tried to solve. Therefore, I put few efforts into making a performant tool at the time.
+
+With version 2, performance was a focus, and the results are really good. Parsed files are now cached, so the initial run is still a bit slow, but subsequent runs are faster by several times.
+
+There is also a **watch mode** where the changes feel instantaneous.
+
+And I am sure we can do much better for future versions! The work done here should help pave the way to having editor support.
+
+## Configuring exceptions
+
+When you enabled a rule in version 1, you wouldn't able to ignore any errors that it reported. You had to edit or fork the rule to ignore the cases you wanted to ignore.
+
+The idea behind that was to avoid ignoring errors locally through a comment of some sort like what happens all over the place with `ESLint` ([which leads to all sorts of problems](https://github.com/jfmengels/elm-review/#is-there-a-way-to-ignore-an-error-or-disable-a-rule-only-in-some-locations)), and instead to have users [think on whether enabling a rule is a good idea](https://github.com/jfmengels/elm-review/#when-to-write-or-enable-a-rule) in the first place. And I still stand by these choices!
```

Comment:
```suggestion
The idea was to avoid ignoring errors locally through a comment of some sort, like what happens all over the place with `ESLint` ([which leads to all sorts of problems](https://github.com/jfmengels/elm-review/#is-there-a-way-to-ignore-an-error-or-disable-a-rule-only-in-some-locations)). Instead users should [think on whether enabling a rule is a good idea](https://github.com/jfmengels/elm-review/#when-to-write-or-enable-a-rule) in the first place. And I still stand by these choices!
```